hadoop-ozone/dist/src/main/compose/ozoneblockade/docker-config

```diff
@@ -34,6 +34,8 @@ OZONE-SITE.XML_hdds.scm.wait.time.after.safemode.exit=30s
 OZONE-SITE.XML_hdds.scm.replication.thread.interval=6s
 OZONE-SITE.XML_hdds.scm.replication.event.timeout=10s
 OZONE-SITE.XML_dfs.ratis.server.failure.duration=35s
+OZONE-SITE.XML_hdds.scm.safemode.min.datanode=3
```

Comment:
This is different because it's used by the python script from `hadoop-ozone/fault-injection-test/network-tests/src/test/blockade` which scales up the docker-compose cluster. In this case we need full cluster.